ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/ServiceMapperFactory.java

```diff
@@ -84,10 +86,12 @@ public class ServiceMapperFactory {
     private final HateoasMapper hateoasMapper;
     private final InterceptorPipeline interceptorPipeline;
     private final MetaInfoMapper metaInfoMapper;
+    private final OpenAPIExampleMapper exampleMapper;
```

Comment:
The `MetaInfoMapper` also maps additional things such as summary, tags, and operation ID. I prefer to have it as a different mapper. But since both of them can modify the request body example, we need to decide on which mapper gets the priority. IMO it should be the `OpenAPIExampleMapper`

Reply:
+1 yeah, I also prefer, If we can handle all example details under the one component. will have a discuss